source/common/formatter/substitution_format_utility.cc

```diff
@@ -85,6 +85,15 @@ void SubstitutionFormatUtils::truncate(std::string& str, absl::optional<size_t>
   }
 }
 
+absl::string_view SubstitutionFormatUtils::truncateStringView(absl::string_view str,
+                                                              absl::optional<size_t> max_length) {
+  if (!max_length) {
```

Comment:
nit: `!max_length.has_value()` prefer the explicit calling

Reply:
The existing function `SubstitutionFormatUtils::truncate()` also uses `if (!max_length)`.

Reply:
It's old code, so, basically we won't change it. It would be better to apply the rule to new added code.

But anyway, this is nit comment.